services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/StepIntersection.java

```diff
@@ -146,6 +147,28 @@ public Point location() {
   @SerializedName("geometry_index")
   public abstract Integer geometryIndex();
 
+  /**
+   * A boolean indicating whether the road exiting the intersection is considered to be in an urban
+   * area. This value is determined by the density of the surrounding road network.
+   * Only available on the {@link DirectionsCriteria#PROFILE_DRIVING} profile.
+   *
+   * @return a value indicating whether the road exiting the intersection is in an urban area
+   */
+  @Nullable
+  @SerializedName("is_urban")
+  public abstract Boolean isUrban();
+
+  /**
+   * An object containing detailed information about the road exiting the intersection along the
+   * route. Properties in this object correspond to properties in the {@link #classes()}
+   * specification. Only available on the {@link DirectionsCriteria#PROFILE_DRIVING} profile.
+   *
+   * @return an object containing detailed road information.
+   */
+  @Nullable
+  @SerializedName("mapbox_streets_v8")
+  public abstract Street streets();
```

Comment:
```suggestion
  public abstract MapboxStreetsV8 mapboxStreetsV8();
```

This value corresponds to streets v8 sources, so I think we need to be explicit here, even if it looks ugly.

Reply:
fixed